src/components/ReservedRockets.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ListGroup } from 'react-bootstrap';
+
+const ReservedRockets = ({ reservedRockets }) => (
+  <div>
+    {reservedRockets.length === 0 ? (
+      <p style={{ color: 'red' }} data-testid="pId">No Reserved Rockets</p>
```

Comment:
- I like how you are displaying `No Reserved Rockets` when there are no reserved rockets. Great job. :clap: